.travis.yml

```diff
@@ -8,10 +8,47 @@ rust:
 cache: cargo
 addons:
   postgresql: '9.5'
+before_install:
+ # install sqlite3 version 3.25.2
+  - |
+    if [[ "$BACKEND" == sqlite ]]; then
+      (sudo apt-get update) &&
+      (wget --quiet -c https://www.sqlite.org/2018/sqlite-autoconf-3250200.tar.gz) &&
+      (tar zxf sqlite-autoconf-3250200.tar.gz;) &&
+      (cd sqlite-autoconf-3250200; \
+       CFLAGS="$CFLAGS -O2 -fno-strict-aliasing \
+                   -DSQLITE_DEFAULT_FOREIGN_KEYS=1 \
+                   -DSQLITE_SECURE_DELETE \
+                   -DSQLITE_ENABLE_COLUMN_METADATA \
+                   -DSQLITE_ENABLE_FTS3_PARENTHESIS \
+                   -DSQLITE_ENABLE_RTREE=1 \
+                   -DSQLITE_SOUNDEX=1 \
+                   -DSQLITE_ENABLE_UNLOCK_NOTIFY \
+                   -DSQLITE_OMIT_LOOKASIDE=1 \
+                   -DSQLITE_ENABLE_DBSTAT_VTAB \
+                   -DSQLITE_ENABLE_UPDATE_DELETE_LIMIT=1 \
+                   -DSQLITE_ENABLE_LOAD_EXTENSION \
+                   -DSQLITE_ENABLE_JSON1 \
+                   -DSQLITE_LIKE_DOESNT_MATCH_BLOBS \
+                   -DSQLITE_THREADSAFE=1 \
+                   -DSQLITE_ENABLE_FTS3_TOKENIZER=1 \
+                   -DSQLITE_MAX_SCHEMA_RETRY=25 \
+                   -DSQLITE_ENABLE_PREUPDATE_HOOK \
+                   -DSQLITE_ENABLE_SESSION \
+                   -DSQLITE_ENABLE_STMTVTAB \
+                   -DSQLITE_MAX_VARIABLE_NUMBER=250000" \
+       ./configure --prefix=/usr \
+                   --enable-threadsafe \
+                   --enable-dynamic-extensions \
+                   --libdir=/usr/lib/x86_64-linux-gnu \
+                   --libexecdir=/usr/lib/x86_64-linux-gnu/sqlite3) &&
+      (cd sqlite-autoconf-3250200; sudo make; sudo make install)
+    fi
 before_script:
   - pip install 'travis-cargo<0.2' --user
   - export PATH=$HOME/.local/bin:$PATH
   - mysql -e "create database diesel_test; create database diesel_unit_test; grant all on \`diesel_%\`.* to 'travis'@'%';" -uroot
+  - sudo chown travis:travis /home/travis/.gitconfig
```

Comment:
It changes it's owner ship to root:root. 
I think this is a bug by TravisCI or git.